app/src/main/java/com/eveningoutpost/dexdrip/Models/Sensor.java

```diff
@@ -158,6 +158,23 @@ public static Sensor getByTimestamp(double started_at) {
                 .executeSingle();
     }
 
+    public static Sensor restartSensor(String sensorUuid) {
```

Comment:
Would this function get repeatedly triggered by the buggy version of manufacturer app?

If so can you wrap it with a rate limit using `pratelimit` as it initiates network activity potentially and we don't want a buggy app to be able to cause that problem internally.

Reply:
Why are we restarting the sensor every 5 minutes?

Reply:
We are not, but they are sending us different sensorId with each broadcast (which is error). We could wait for their fix and then merge...

Reply:
`pratelimit` is now used.